src/sentry/integrations/vsts/integration.py

```diff
@@ -162,25 +163,38 @@ def get_pipeline_views(self):
 
     def build_integration(self, state):
         data = state['identity']['data']
+        oauth_data = self.get_oauth_data(data)
         account = state['account']
         instance = state['instance']
         user = get_user_info(data['access_token'])
+        subscription_id, subscription_secret = self.create_subscription(
+            instance, account['AccountId'], oauth_data)
         scopes = sorted(VSTSIdentityProvider.oauth_scopes)
 
         return {
             'name': account['AccountName'],
             'external_id': account['AccountId'],
             'metadata': {
                 'domain_name': instance,
+                'scopes': scopes,
+                'subscription_id': subscription_id,
+                'subscription_secret': subscription_secret,
```

Comment:
How will two organizations creating the same integrations affect this? I assume we only want one subscription per integration? I can imagine this will cause problems there.

Reply:
> I assume we only want one subscription per integration?
I think so, too.

Reply:
I think that's probably going to need to be a separate PR. this is getting a bit much for me as is. I'll follow this up with something to address that issue.

Reply:
I don't know if I follow, actually. Each time someone sets up the VSTS integration, we'll create a new Subscription. It seems like the way this is working is what we want, unless I'm missing something.